app/components/primer/beta/button.pcss

```diff
@@ -52,10 +52,6 @@
     cursor: not-allowed;
     box-shadow: none;
   }
-
-  /* &:focus {
-    @mixin focusOutline;
-  } */
```

Comment:
with this commented out, PCSS copies it as a literal comment with no parsing into the final css output. this was causing `test/components/component_css_test.rb` to fail in the test suite, because that test looks for the presence of any `&` regardless of context